api/agent/drivers/docker/docker.go

```diff
@@ -276,6 +275,7 @@ func (drv *DockerDriver) PrepareCookie(ctx context.Context, c drivers.Cookie) er
 		return err
 	}
 
+	cookie.opts.Context = ctx
```

Comment:
switching to correct ctx here, ctx passed in CreateCookie() should not be used in CreateContainer().